assets/src/scss/blocks/ActionsList/ActionsListStyle.scss

```diff
@@ -80,8 +80,9 @@
 
     .read-more-nav {
       display: flex;
-      justify-content: flex-end;
-      margin-top: 25%;
+      position: absolute;
+      bottom: 0;
+      right: 0;
```

Comment:
For RTL screens it should be aligned to the left instead

Reply:
Nice catch!
I think this is the first time in my life I have to consider an RTL screen.

Reply:
Testing page: https://www-dev.greenpeace.org/test-pluto/actions-list-test-page/

Reply:
We have a few NROs that use RTL languages, so we need to think about it 🙂